src/test/java/com/google/sps/servlets/ScheduledInterviewServletTest.java

```diff
@@ -374,7 +378,7 @@ public void onlyReturnsInterviewersWithMatchingCompanyAndJob() throws IOExceptio
                 Instant.parse("2020-07-20T12:45:00Z"), Instant.parse("2020-07-20T13:45:00Z")),
             googleSWE1.id(),
             emailToId("user@company.org"),
-            "meet_link",
+            String.valueOf(actual.get(0).id()),
```

Comment:
Why is this like this?

Reply:
The mock meetlink is the id of the scheduled interview as a string

Reply:
Ok, that's fine then.